tests/Unit/PointwiseFunctions/AnalyticSolutions/Xcts/SchwarzschildKerrSchildIsotropic.py

```diff
@@ -0,0 +1,209 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+from numpy import sqrt, exp
+from scipy.optimize import newton
+
+# Areal Kerr-Schild coordinates
+
+
+def lapse_areal(r, mass):
+    return 1. / sqrt(1. + 2. * mass / r)
+
+
+def lapse_areal_deriv(r, mass):
+    return lapse_areal(r, mass)**3 * mass / r**2
+
+
+def shift_magnitude_areal(r, mass):
+    return 2. * mass * lapse_areal(r, mass)**2 / r
+
+
+def shift_areal(x, mass):
+    r = np.linalg.norm(x)
+    return shift_magnitude_areal(r, mass) * x / r
+
+
+def spatial_metric_areal(x, mass):
+    r = np.linalg.norm(x)
+    return np.identity(3) + 2. * mass * np.outer(x, x) / r**3
+
+
+def extrinsic_curvature_trace_areal(r, mass):
+    lapse = lapse_areal(r, mass)
+    return 2. * mass * lapse**3 / r**2 * (1. + 3. * mass / r)
+
+
+def extrinsic_curvature_trace_areal_deriv(r, mass):
+    lapse = lapse_areal(r, mass)
+    K = extrinsic_curvature_trace_areal(r, mass)
+    return (K / r * (3. * mass * lapse**2 / r - 3. * mass /
+                     (r + 3. * mass) - 2.))
```

Comment:
I'm not sure if I'm missing something, but according to the cpp implementation, shouldn't this be divided by `square(rbar)`?